### DISCUSSION TITLE
Map Toolbar Settings Flyout Too Small

### DISCUSSION THREAD

**Discussion by dmmagic** [encounterplus/encounterplus#492] [Ideas]
When the gear icon on the map toolbar is pressed, the flyout menu opens up pretty small -- and there's a lot of info presented that needs to be scrolled through. Can the menu open up much taller? Maybe the entire height of the map toolbar?

**Comment by jurex** [comment]
bumped height of the settings window to 400 units on ipad. Don't want to cover the entire screen tho, so let's see how it goes like this. closing for now
